css/style.css

```diff
@@ -487,6 +487,46 @@ li {
   cursor: not-allowed;
 }
 
+/**** menu ****/
+
+.menu {
+  position: fixed;
+  top: 0;
+  left: 0;
+  width: 100%;
+  height: 100%;
+  background: #6070ffe3;
+  backdrop-filter: blur(5px);
+  z-index: 5000;
+}
```

Comment:
Hi @aamirkhan2478!
Thanks for your review, yes, I was asked to change the color when I was doing the accessibility exercise because I didn't have enough contrast between the color of the letters and the background in the original Figma design. 
 Before that exercise my design was identical. 
 Do you suggest I revert the changes?